portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -14,46 +14,43 @@
-        UserService userService = UserServiceFactory.getUserService();
-        PrintWriter out = response.getWriter();
-
-        if (!userService.isUserLoggedIn()) {
-            String loginUrl = userService.createLoginURL("/Login");
-            out.println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
-        } else {
-            String logoutUrl = userService.createLogoutURL("/");
-            out.println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
-            out.println("<p>Username:");
-            out.println("<form method=\"POST\" action=\"Login\">");
-            out.println("<input name=\"username\" >");
-            out.println("<br />");
-            out.println("<button>Submit</button>");
-            out.println("</form>");
-        }
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    PrintWriter out = response.getWriter();
+
+    if (!userService.isUserLoggedIn()) {
+      String loginUrl = userService.createLoginURL("/Login");
+      out.println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
+    } else {
+      String logoutUrl = userService.createLogoutURL("/");
+      out.println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+      out.println("<p>Username:");
+      out.println("<form method=\"POST\" action=\"login\">");
+      out.println("<input name=\"username\" >");
+      out.println("<br />");
+      out.println("<button>Submit</button>");
+      out.println("</form>");
     }
+  }
 
-    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        UserService userService = UserServiceFactory.getUserService();
-        if (!userService.isUserLoggedIn()) {
-            response.sendRedirect("/Login");
-            return;
-        }
-        String username = request.getParameter("username");
-        System.out.println("username:================");
-        System.out.println(username);
-        String id = userService.getCurrentUser().getUserId();
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    if (!userService.isUserLoggedIn()) {
+      response.sendRedirect("/login");
+      return;
+    }
+    String username = request.getParameter("username");
+    String id = userService.getCurrentUser().getUserId();
 
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-        Entity newUser = new Entity("user", id);
-        newUser.setProperty("id", id);
-        newUser.setProperty("username", username);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity newUser = new Entity("user", id);
+    newUser.setProperty("id", id);
+    newUser.setProperty("username", username);
```

Comment:
Same comment here about static constants.  All of these parameter/field names could be representing as constants either in this file or in a common file/class that all servlets could use to preserve consistency.

Reply:
Same as above!

Reply:
Made a key class pt. 2